Dockerfile.agnos

```diff
@@ -211,9 +213,8 @@ RUN echo -n "10.1" > /VERSION
 # ################## #
 # #### Cleaunup #### #
 # ################## #
-
 RUN rm -rf /usr/share/icons/* && \
     rm -rf /var/lib/apt/lists/* && \
     rm -rf /home/$USERNAME/.cache && \
     rm -rf /root/.cache && \
-    apt-get clean
+    apt-get clean
```

Comment:
Also the newline should probably be preserved. (I guess its a setting of your IDE/text editor). Same for the other files where this happened.

Reply:
done.